templates/web/base/dashboard/index.html

```diff
@@ -152,31 +74,85 @@ <h2>[% loc('Reports') %]</h2>
         <label for="end_date">[% loc('End Date') %]</label>
         <input name="end_date" type="date" value="[% end_date | html %]" class="form-control">
     </p>
-    <p class="no-label">
-        <input type="submit" class="btn" value="[% loc('Look up') %]">
-        <a class="btn export_as_csv" href="[% c.req.uri_with({ export => 1 }) %]">[% loc('Export as CSV') %]</a>
-    </p>
 </div>
 
-<table width="100%" id="reports">
+<p align="center">
+    <input type="hidden" name="body" value="[% body.id | html %]">
+    <input type="submit" class="btn" value="[% loc('Look up') %]">
+    <input type="submit" class="btn" name="export" value="[% loc('Export as CSV') %]">
+</p>
+
+</form>
+
+<p>
+Within the specified timeframe:
+[% tprintf( loc('%d opened, %d closed, %d fixed'), open, closed, fixed ) %]
+</p>
+
+<p>
+Group by:
+<a href="[% c.uri_with({ group_by => 'category' }) %]">category</a>
+| <a href="[% c.uri_with({ group_by => 'state' }) %]">state</a>
+| <a href="[% c.uri_with({ group_by => 'month' }) %]">month</a>
+| <a href="[% c.uri_with({ group_by => 'category+state' }) %]">category+state</a>
+| <a href="[% c.uri_with({ group_by => 'device/site' }) %]">device/site</a>
+</p>
+
+Current state of reports that match filters:
+
+<table width="100%" id="overview">
+  [% FOR k IN grouped.keys.sort %]
```

Comment:
This sort order doesn't really make sense for anything other than category names. It's particularly unhelpful for months.

Reply:
Gah, doh. Sorting states is also what current admin/stats does, so I'm not too bothered about keeping that, though I agree bunching open/fixed/closed together or similar would be better. Have to do something about months though, yes! So might try and sort both at once.